### PR TITLE
Refactored Code

### PR DESCRIPTION
**Pull request by rishimadan14**

### CODE REVIEW

File: api/index.py [hunk 1]
```diff
@@ -13,44 +12,48 @@
...
-    if file.filename == '':
+    if file.filename == '':#empty filename 
         return jsonify({"error": "No selected file"}), 400
 
-    if not allowed_file(file.filename):
+    if not allowed_file(file.filename):#extension checking
         return jsonify({"error": "Invalid file format"}), 400
 
-    filename = "data.csv"  
-
```
Comment (aniruddh622003) [inline api/index.py:37]: I think you removed the functionality of saving file in 'data.csv'

**Review by aniruddh622003 — CHANGES_REQUESTED** [review]

**Comment by rishimadan14** [comment]
sharing the changes for saving filename as data.csv every time a new file hits.
    # in order to track harmful content
    filename = secure_filename("data.csv")
    file_path = os.path.join(app.config['Upload_Folder'], filename)

**Comment by rishimadan14** [comment]
changes also pulled

**Comment by aniruddh622003** [comment]
changes push to krde bhai

**Review by aniruddh622003 — APPROVED** [review]

**Pull request merged**